lib/dotcom_web/views/schedule/timetable.ex

```diff
@@ -3,6 +3,7 @@ defmodule DotcomWeb.ScheduleView.Timetable do
   alias DotcomWeb.ViewHelpers, as: Helpers
   alias DotcomWeb.PartialView.SvgIconWithCircle
   alias Stops.Stop
+  alias Routes.Route
```

Comment:
Is this used?

Reply:
uh yeah, by the types in the last function